packages/terra-menu/src/_MenuContent.jsx

```diff
@@ -123,6 +121,8 @@ class MenuContent extends React.Component {
     this.validateFocus = this.validateFocus.bind(this);
     this.needsFocus = props.isFocused;
     this.handleContainerRef = this.handleContainerRef.bind(this);
+    this.menuHeaderId = `terra-menu-headertitle-${uuidv4()}`;
```

Comment:
Hard-coded outside of the component `menuHeaderId` and `menuTopHeaderId` were preventing jest.spyOn.mockImplementation work in Menu.test.jest. 
This change is consistent with other terra element's like this: https://github.com/cerner/terra-framework/blob/3ccdc22e9a8a13014688dcf13aa667bda90cff07/packages/terra-date-input/src/DateInput.jsx#L137
This change is not user-facing and doesn't need to be reflected in changelog.